scripts/linuxDebRename.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const { rename } = require('fs').promises;
```

Comment:
There is already a lot of code that is in [macosPackRename.js](https://github.com/asyncapi/cli/blob/master/scripts/macosPackRename.js). Maybe merge both, and have one script to rename all distros? and `"pack:macos:rename": "node scripts/macosPackRename.js"` becomes `"pack:rename": "node scripts/releasePackagesRename.js"` ?